### DISCUSSION TITLE
armed / unarmed.

### DISCUSSION THREAD

**Discussion by Q-889** [akaAgar/briefing-room-for-dcs#229] [Ideas]
Greetings guys,

I think it will be a cool Idea if I also can generate a Practice mission too, A/A A/G Moving target the ability to make the enemy armed and unarmed. But as the main principle is generating a mission it will be great if we have the ability now to make the Ai armed or unarmed.

Thanks for the amazing work guys.

**Comment by akaAgar** [comment]
That's a great idea, I'll add it as an option on the next version
